tests/roles/run_tests/templates/cluster-template-workers-kubeadm-config-centos.yaml

```diff
@@ -1,8 +1,10 @@
 # CentOS specific worker kubeadm config
 preKubeadmCommands:
   - systemctl restart NetworkManager.service
-  - nmcli connection load /etc/sysconfig/network-scripts/ifcfg-eth0
-  - nmcli connection up filename /etc/sysconfig/network-scripts/ifcfg-eth0
+  - nmcli con add load /etc/NetworkManager/system-connections/eth0.nmconnection
+  - nmcli connection up filename /etc/NetworkManager/system-connections/eth0.nmconnection
+  - nmcli con add load /etc/NetworkManager/system-connections/{{ IRONIC_ENDPOINT_BRIDGE }}.nmconnection
+  - nmcli connection up filename /etc/NetworkManager/system-connections/{{ IRONIC_ENDPOINT_BRIDGE }}.nmconnection
```

Comment:
```suggestion
  - nmcli connection load /etc/NetworkManager/system-connections/eth0.nmconnection
  - nmcli connection up eth0
  - nmcli connection load /etc/NetworkManager/system-connections/{{ IRONIC_ENDPOINT_BRIDGE }}.nmconnection
  - nmcli connection up {{ IRONIC_ENDPOINT_BRIDGE }}
```

Reply:
Yes you are right. I somehow missed it. Changed it